packages/babel-plugin-open-source/babel.js

```diff
@@ -30,18 +33,46 @@ module.exports = declare(api => {
       if (process.env.NODE_ENV !== 'development') return
 
       const location = path.container.openingElement.loc
+      let url = null;
+      let editor = state.opts?.editor?.toLowerCase() || 'vscode';
 
       // the element was generated and doesn't have location information
       if (!location) return
 
+      try {
+        if(state.opts.envPath) {
```

Comment:
Feels a bit odd to ask for an `envPath`. Is it safe to follow the popular convention of finding a `.env` file at the root of the repository?

Reply:
Sidenote: something to consider: Is `.editorconfig` the right place to add our custom key? (https://editorconfig.org)

Reply:
Yeah I can try the root .env approach.

For `.editorconfig`, it is not personalized right?
I mean it is pushed the GitHub so it's not unique..

Whereas .env aren't pushed, hence they can contain unique values..

Makes sense? 😅

Reply:
Makes sense!

didn't know .editorConfig are pushed to github 👍

Reply:
Even I am not sure if they are supposed to be pushed.
I have just noticed them in some places on GitHub.